root/static/scripts/common/components/DescriptiveLink.js

```diff
@@ -0,0 +1,42 @@
+// This file is part of MusicBrainz, the open internet music database.
+// Copyright (C) 2015—2016 MetaBrainz Foundation
+// Licensed under the GPL version 2, or (at your option) any later version:
+// http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
```

Comment:
So e.g. with something like `<EntityLink entity={entity} {...props} />` below, the JSX gets converted into `React.createElement(EntityLink, {entity: entity, ...})` during compilation. You gotta import React everywhere you use JSX, basically.